src/Object/Metadata.php

```diff
@@ -0,0 +1,89 @@
+        $this->domain = $domain;
+        $this->options = $options;
+    }
+
+    public function getTitle(): string
+    {
+        return $this->title;
+    }
+
+    public function getDescription(): ?string
+    {
+        return $this->description;
+    }
+
+    public function getImage(): ?string
+    {
+        return $this->image;
+    }
+
+    public function getDomain(): ?string
+    {
+        return $this->domain;
+    }
+
+    public function getOptions(): array
+    {
+        return $this->options;
+    }
+
+    public function getOption($name, $default = null)
```

Comment:
String $name ?

Reply:
No, https://github.com/sonata-project/SonataAdminBundle/pull/5399#discussion_r247314113

Reply:
You would have to change the interface, and that would be a BC-break